values-nginx.yaml

```diff
@@ -50,13 +128,13 @@ ingress:
     # ingress.kubernetes.io/tls-acme: "true"
     kubernetes.io/ingress.class: nginx
     nginx.ingress.kubernetes.io/ssl-redirect: "false"
-    nginx.ingress.kubernetes.io/rewrite-target: /$2
+    nginx.ingress.kubernetes.io/rewrite-target: /
     nginx.ingress.kubernetes.io/affinity: cookie
     nginx.ingress.kubernetes.io/session-cookie-name: JSESSIONID
     nginx.ingress.kubernetes.io/proxy-connect-timeout: "60"
     nginx.ingress.kubernetes.io/proxy-read-timeout: "60"
     nginx.ingress.kubernetes.io/proxy-send-timeout: "60"
-  path: /xl-release(/|$)(.*)
```

Comment:
Are we removing path based routing?

Reply:
For Keycloak, yes. After the "path" issue is fixed will update the ingress resource.